src/advanced-settings/AdvancedSettings.jsx

```diff
@@ -0,0 +1,188 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  Container,
+  Button,
+  Layout,
+  useToggle,
+} from '@edx/paragon';
+import {
+  CheckCircle, Info, WarningFilled,
+} from '@edx/paragon/icons';
+import { FormattedMessage, injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import { fetchCourseAppSettings, updateCourseAppSetting, fetchProctoringExamErrors } from './data/thunks';
+import { getCourseAppSettings, getSavingStatus, getProctoringExamErrors } from './data/selectors';
+import SettingCard from './setting-card/SettingCard';
+import SettingAlert from './setting-alert/SettingAlert';
+import SettingsSidebar from './settings-sidebar/SettingsSidebar';
+import { RequestStatus } from '../data/constants';
+import { parseArrayOrObjectValues } from '../utils';
+import messages from './messages';
+
+const AdvancedSettings = ({ intl, courseId }) => {
+  const advancedSettingsData = useSelector(getCourseAppSettings);
+  const savingStatus = useSelector(getSavingStatus);
+  const proctoringExamErrors = useSelector(getProctoringExamErrors);
+  const dispatch = useDispatch();
+  const [showWarningAlert, setShowWarningAlert] = useState(false);
+  // eslint-disable-next-line no-unused-vars
+  const [isOn, setOn, setOff, toggle] = useToggle(false);
+  const [settingValues, setSettingValues] = useState({});
```

Comment:
let's rename this to `editedSettings`, so it will be more obvious what is stored in this state